apps/namadillo/src/App/Masp/MaspShield.tsx

```diff
@@ -165,8 +167,8 @@ export const MaspShield: React.FC = () => {
           isShielded: true,
         }}
         feeProps={feeProps}
-        isSubmitting={isPerformingTransfer}
-        errorMessage={generalErrorMessage}
+        isSubmitting={isPerformingTransfer || isSuccess}
```

Comment:
I'm just reading this semantically, not checking the implementation, but looks strange `isSubmitting === isSuccess`

Should it be renamed to `disabled={isPerformingTransfer || isSuccess}` ?

Reply:
Maybe we need to rephrase this, but we need to keep the submitting animation running while we have the tx watcher verifying if the transaction was applied on chain. The execution on useTransfer only handles the broadcasting, so we keep it submitting while we make this verification. I'll try to improve this

Reply:
hmm it gives me one idea, if we could add the pooling of the wrapper/tx to useTransfer, then we'll have the applied status used for all transactions, not only transfers

Reply:
I don't think it's a good idea cause this way we won't be able to track transactions when users leave the page :/